Week2/homework/maartjes_work.js

```diff
@@ -45,3 +45,10 @@ const tuesday = [
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+const salary = tasks
+  .map((task) => (task.duration / 60))
+  .filter((duration) => duration >= 2)
+  .map((duration) => duration * 23)
+  .reduce((acc, current) => acc + current);
+
+console.log('Maartje has earned €' + salary.toFixed(2));
```

Comment:
you could consider to pass the tasks and salary as two arguments, something like:


const calculateEarnings = (arrayOfTasks, hourlyWage) =>
arrayOfTasks.map((task) => task.duration / 60)
.filter((hoursAmount) => hoursAmount >= 2)
.map((paidHoursAmount) => paidHoursAmount * hourlyWage)
.reduce((sum, amount) => sum + amount, 0)
.toFixed(2);